xbmc/ServiceManager.cpp

```diff
@@ -23,9 +23,12 @@
 #include "utils/log.h"
 #include "interfaces/AnnouncementManager.h"
 #include "interfaces/generic/ScriptInvocationManager.h"
+#include "addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h"
 
 bool CServiceManager::Init1()
 {
+  V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```

Comment:
Is a double use like
```cpp
V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
V2::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```
after level 2 comes in OK for you?

I think this faults becomes fixed with level 3 where everything on add-on's becomes performed with classes instead of `C` functions.

For all who not know, see: https://github.com/AlwinEsch/pvr.demo/commit/a213319e9c267ced38cccacecce25e9c51235bde